### PR TITLE
Added option to ignore a channel

### PR DESCRIPTION
**Pull request by VoidNulll**
Updated index, readme, and template config.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +26,8 @@ Enter your bot token and all correct informations:
             "guildID": "111111",
             "channelID": "121212",
             "whID": "232323",
-            "whToken": "webhookToken2323"
+            "whToken": "webhookToken2323",
+            "ignore": false
```
Comment (Khaazz) [inline README.md]: typo here as you named it ignore and not ignored

Comment (VoidNulll) [reply README.md]: Not a typo. I made it `ignore` to have it make more sense to the end user.

Comment (Khaazz) [reply README.md]: faire enough.Indeed my bad. However it's called ignored in the template.

**Comment by VoidNulll** [comment]
Updated!

**Pull request merged**